plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/toolwindow/OuterAmazonQPanel.kt

```diff
@@ -0,0 +1,33 @@
+// Copyright 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.amazonq.toolwindow
+
+import com.intellij.openapi.components.Service
+import com.intellij.openapi.components.service
+import com.intellij.openapi.project.Project
+import com.intellij.ui.components.panels.Wrapper
+import com.intellij.util.ui.components.BorderLayoutPanel
+import software.aws.toolkits.core.utils.getLogger
+import javax.swing.JComponent
+
+@Service(Service.Level.PROJECT)
```

Comment:
not sure why the panels need to be services

Reply:
We need single instances of panels, the inner contents change

Reply:
the panel is only referenced in AmazonQToolWindowFactory

Reply:
Yeah, passing the same instance to all places now